Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/README.md

```diff
@@ -1054,6 +1054,226 @@ Gets compliance search action from the Security & Compliance Center.
+| description | Case description. | Optional | 
+| external_id | Case external Id. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365.SecurityAndCompliance.ComplianceCase.Name | String | Case name. | 
+| O365.SecurityAndCompliance.ComplianceCase.Status | String | Case status. | 
+| O365.SecurityAndCompliance.ComplianceCase.CreatedDateTime | String | Case created date time. | 
+
+### o365-sc-compliance-case-list
+
+***
+List different types of compliance cases in the Microsoft Purview compliance portal
+
+#### Base Command
+
+`o365-sc-compliance-case-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| identity | List cases by identity. | Optional | 
+| case_type | List cases by type. Possible values are: AdvancedEdiscovery, ComplianceClassifier, DataInvestigation, DSR, eDiscovery, InsiderRisk, SupervisionPolicy. | Optional | 
+| limit | Limit returned cases list size. Default is 50. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365.SecurityAndCompliance.ComplianceCase.Name | String | Case name. | 
+| O365.SecurityAndCompliance.ComplianceCase.Status | String | Case status. | 
+| O365.SecurityAndCompliance.ComplianceCase.GUID | UUID | Case GUID. | 
+| O365.SecurityAndCompliance.ComplianceCase.CreatedDateTime | String | Case created date time. | 
+
+### o365-sc-compliance-case-delete
+
+***
+Removes compliance cases from the Microsoft Purview compliance portal or the Microsoft Purview compliance portal.
+
+#### Base Command
+
+`o365-sc-compliance-case-delete`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| identity | Delete case by identity. | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+### o365-sc-case-hold-policy-create
+
+***
+Creates new case hold policies in the Microsoft Purview compliance portal
+
+#### Base Command
+
+`o365-sc-case-hold-policy-create`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| policy_name | Name of the policy to create. | Required | 
+| case | eDiscovery case, Case Name, Case Identity (GUID value). | Required | 
```

Comment:
Not clear.  Do I need to give all that information or just one of them?

Reply:
Perhaps it should be as follows:
The eDiscovery case, case name, or case identity (GUID value).